step.yml

```diff
@@ -119,19 +119,29 @@ inputs:
       description: |-
         Root directory of your Cordova project, where your Cordova config.xml exists.
       is_required: true
-  - options: --buildFlag="-UseModernBuildSystem=0"
+  - options:
     opts:
       title: "Options to append to the cordova-cli build command"
       description: |-
         Use this input to specify custom options, to append to the end of the cordova-cli build command.
 
         The new Xcode build system is now supported in cordova-ios@5.0.0 (https://github.com/apache/cordova-ios/issues/407).
```

Comment:
It's safe to delete this sentence too from the description.

Reply:
Yes it's safe because, it's only default "text" in input in this option, with new option you've a new system in default.